cc/imgproc/imgprocBindings.h

```diff
@@ -204,6 +204,95 @@ namespace ImgprocBindings {
 		  };
 	  };
   };
+
+	class Accumulate : public CvClassMethodBinding<Mat> {
+	public:
+		cv::Mat dst;
+		
+		void createBinding(std::shared_ptr<FF::Value<cv::Mat>> self) {
+			auto src = req<Mat::Converter>();
+			auto mask = opt<Mat::Converter>("mask", cv::noArray().getMat());
+			executeBinding = [=]() {
+				auto depth = self->ref().depth();
+				if (depth != CV_32F && depth != CV_64F)
+					throw std::runtime_error("dst must have a depth of CV_32F or CV_64F - mat depth:" + std::to_string(depth));
```

Comment:
I think an exception will already be thrown by OpenCV, so this check is not necessary or am I wrong?

Reply:
dst must be same number of channels and size as src and has a depth of CV_32F or CV_64F
opencv throws exception for all cases.
but depth exception message is unclear. so i added the check code.

error msg on wrong size and channel >
 Assertion failed (_src.sameSize(_dst) && dst == scn) in cv::accumulate, file ~

error msg on wrong depth >
 Assertion failed (func != 0) in cv::accumulate, file ~